docs/sources.md

```diff
@@ -0,0 +1,268 @@
+# Sources
```

Comment:
I’m feeling this should be placed in a directory like /docs/internal/, as this isn’t expected to be read by end users

Reply:
I'm fine with moving this there. Let's get all of the missing content in first.